packages/components/src/Spinner/Spinner.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import classnames from "classnames";
+import styles from "./Spinner.css";
+
+export interface SpinnerProps {
+  readonly size?: "small" | "base";
```

Comment:
We've been trying to add a little docs line explaining each the props as well as any defaults here.

eg.
```suggestion
  /**
   * Specifies the size of the spinner.
   *
   * @default "base"
   */
  readonly size?: "small" | "base";
```

Reply:
Excellent, I've done this with the rest of the props as well!